repose-aggregator/docs/src/asciibinder/welcome/release-notes.adoc

```diff
@@ -21,6 +21,7 @@
 * [.new-tag]#New# | https://repose.atlassian.net/browse/REP-7600[REP-7600] - Added the <<../filters/openapi-validator.adoc#, OpenApi Validator Filter>>.
 * [.bug-tag]#Bug Fix# | https://repose.atlassian.net/browse/REP-7608[REP-7608] - Fixed bug where *Repose* was acting like it was sending content when the body was empty and chunked encoding was on.
 * [.bug-tag]#Bug Fix# | https://repose.atlassian.net/browse/REP-7663[REP-7663] - Fixed a file contention issue between Repose processes in the EAR unpacking logic.
+* [.new-tag]#New# | https://repose.atlassian.net/browse/REP-7600[REP-7600] | https://repose.atlassian.net/browse/REP-7789[REP-7789] - Added the <<../filters/openapi-validator.adoc#, OpenApi Validator Filter>> which provides *Swagger/OpenApi* api validation support.
```

Comment:
`api` -> `API`

Also, it looks like the line for `REP-7600` already exists a few lines above this one. Either the other line should be deleted, or this line should be updated to only reflect the changes due to `REP-7789`.